index.js

```diff
@@ -1,4 +1,5 @@
 import { sendAnalyticEvent } from './src/analytics';
 import { APVideoPlayer } from './src/player';
+import { DataSourceService } from './src/datasource/dsService';
```

Comment:
you can do directly `export { DataSourceService } from './src/datasource/dsService'`